src/charactersheet/viewmodels/common/character_picker/index.html

```diff
@@ -1,6 +1,13 @@
 <div class="container-fluid">
   <div class="row" style="margin-top:50px;">
     <div class="col-sm-12 col-md-6 col-md-offset-2 text-center">
+      <div class="alert alert-info alert-dismissible">
+       <small>
+         A new version of Adventurer's Codex is coming! Check
+         <a href="/core/" title="Go to the new beta version of Adventurer's Codex">
+         out the beta now</a>! (You can always go back here).
```

Comment:
Maybe say:
>You can always go back by using this [link]()
and have the link be a url to the original url

Reply:
I was gonna make a PR to the new UI too with a link to the old UI. I figure that'll be easier than having people copy the link we give them here. It's more just for reassurance.